ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/ParcelableMemoryCache.java

```diff
@@ -61,43 +81,108 @@ public int describeContents() {
         @Override
         public void writeToParcel(final Parcel dest, final int flags) {
             dest.writeInt(tokenId);
+            dest.writeString(tag);
         }
 
         @Override
-        public boolean equals(final Object other) {
-            if (this == other) {
+        public boolean equals(final Object o) {
```

Comment:
That's not really an improvement for the name here, of course it's autogenerated.